katas/metal-slug/jeanpeare/Vehicles.js

```diff
@@ -0,0 +1,6 @@
+export class Vehicles {
```

Comment:
Typically, class names should be defined with names in a singular form. In this case, it would be better to use just `Vehicle` instead